File: CPP_programming/Day7_inheritance-1/A1_Employee/Date.cpp
#include"Date.h"

Date::Date()//////////////constructor
{
    date=1;
    month=1;
    year=2000;
}
Date::Date(int d, int m, int y)
{
    date=d;
    month=m;
    year=y;
}
Date::Date(Date &obj)
{
    date = obj.date;
    month = obj.month;
    year = obj.year;
}
void Date::display()///////////////facilitator
{
    cout<<"Date: "<<date<<"/"<<month<<"/"<<year<<"\n";
}













































// Made with ❤️;

File: Core_Java/Day6/src/A1/office/utility/Date.java
package A1.office.utility;

public class Date {
	private int dd;
	private int mm;
	private int yy;
	public Date() {
		super();
	}
	public Date(int dd, int mm, int yy) {
		super();
		this.dd = dd;
		this.mm = mm;
		this.yy = yy;
	}
	public Date(Date d) {
		super();
		this.dd = d.dd;
		this.mm = d.mm;
		this.yy = d.yy;	
	}
	public void display()
	{
		System.out.println("Date: "+dd+"/"+mm+"/"+yy);
	}
	

}
